dtmi/azureiot/phone_as_a_device/phoneasadevice-1.json

```diff
@@ -0,0 +1,175 @@
+{
+    "@id": "dtmi:azureiot:Phone_As_A_Device:PhoneAsADevice;1",
```

Comment:
let's sync offline to provide the @id, some things to consider
- Namespace, aligned with other azure interfaces. Do we want to indicate this is a sample?
- Avoid duplication